api/models/v0.2/modelResponse.js

```diff
@@ -0,0 +1,98 @@
+const url = require('url');
+
+/**
+ * modelPatronCreatorResponse(data, status)
+ * Model the response from creating a new patron.
+ *
+ * @param {object} data
+ * @param {number} status
+ * @return {object}
+ */
+function modelPatronCreatorResponse(responseData, status, requestData) {
+  return {
+    data: [
+      {
+        id: parseInt(responseData.link.split('/').pop()),
```

Comment:
Can you add `10` as the radix value to `parseInt` just for an extra safety catch?

Reply:
Sounds good; I did that here and a few lines below in the other `parseInt`